src/style.css

```diff
@@ -605,8 +605,9 @@ body.nav-active { overflow: hidden; }
   top: 0;
   left: 0;
   width: 100%;
-  background-color: transparent;
-  padding-block: 40px;
+  background-color:var(--smoky-black-2);
```

Comment:
reduce the opacity little, its still bit dark, want lighter version.

Reply:
okayy

Reply:
It looks fine tho no worries please star this  repo 👍🤞.

Reply:
thanks brother

Reply:
@Anu27n please add gssoc-ext and level tag here , score are counted for pr having this tags